config/cola-openai.conf

```diff
@@ -0,0 +1,20 @@
+
+include "cola-elmo.conf"
+
+
+// Training setting
+
+do_target_task_training = 0
+allow_reuse_of_pretraining_parameters = 1
+batch_size = 2
```

Comment:
Why so small?

Reply:
I tried several different batch size, ranging from 2 to 6. 6 is as large as we can afford on k80 but will out-of-memory on smaller gpus. 2 can run on any gpu in prince, and it performs better. Some researches show smaller bs results in better generalization.

Reply:
I've mostly seen papers showing the opposite effect for batch size, but it's okay to leave it in if it works.